src/components/LHNOptionsList/OptionRowLHNData.js

```diff
@@ -41,6 +42,9 @@ const propTypes = {
         }),
     ),
 
+    /** The list of parent report action of the report */
```

Comment:
```suggestion
    /** The actions from the parent report */
```

This is the comment used elsewhere in the app, which I think is clearer and more accurate.